ts/torch_handler/base_handler.py

```diff
@@ -73,6 +81,9 @@ def initialize(self, context):
             self.model = self._load_torchscript_model(model_pt_path)
 
         self.model.eval()
+	if ipex_enabled:
+            self.model = self.model.to(memory_format=torch.channels_last)
+            self.model = ipex.optimize(self.model)
```

Comment:
@min-jean-cho does it choose optimization level 01 by default?

Reply:
Yes, ipex will choose level 'O1' by default.